cypress/integration/finance/create_GL_journal_line_entry_spec.js

```diff
@@ -8,30 +28,27 @@ describe('Creating GL journal entry and change posting type', function() {
   it('create journal', function() {
     cy.clickHeaderNav(Cypress.messages.window.new.caption);
 
-    cy.writeIntoStringField('Description', 'Spende');
-    cy.selectInListField('GL_Category_ID', 'Standard');
-    cy.selectInListField('PostingType', 'Actual');
-    cy.selectInListField('C_Currency_ID', 'CHF');
-    cy.selectInListField('C_DocType_ID', 'Journal');
+    cy.writeIntoStringField('Description', description);
+    cy.selectInListField('GL_Category_ID', category);
+    cy.selectInListField('PostingType', postingType);
+    cy.selectInListField('C_Currency_ID', currency);
+    cy.selectInListField('C_DocType_ID', docType);
     cy.selectDateViaPicker('DateDoc');
   });
 
   it('add new entry line', function() {
     cy.pressAddNewButton();
-    cy.writeIntoStringField('Description', 'Spende', true /*modal*/);
-    cy.writeIntoLookupListField('Account_DR_ID', '10', '1020_UBS Kontokorrent', false /*typeList*/, true /*modal*/);
-    cy.writeIntoStringField('AmtSourceDr', '100.00', true /*modal*/);
-    cy.writeIntoLookupListField('Account_CR_ID', '10', '1020_UBS Kontokorrent', false /*typeList*/, true /*modal*/);
-    cy.selectInListField('C_Activity_ID', '250_Verwaltung', true /*modal*/);
+    cy.writeIntoStringField('Description', description, true /*modal*/);
+    cy.writeIntoLookupListField('Account_DR_ID', account, account, false /*typeList*/, true /*modal*/);
+    cy.writeIntoStringField('AmtSourceDr', amountSource, true /*modal*/);
+    cy.writeIntoLookupListField('Account_CR_ID', account, account, false /*typeList*/, true /*modal*/);
+    cy.selectInListField('C_Activity_ID', activity, true /*modal*/);
     cy.pressDoneButton();
   });
 
   it('complete', function() {
     cy.fixture('misc/misc_dictionary.json').then(miscDictionary => {
```

Comment:
I think `cy.fixture('misc/misc_dictionary.json').then(miscDictionary => {` can also be deleted since we're using `cy.completeDocument();`

Reply:
done